src/relayFeeCalculator/chain-queries/queries.e2e.ts

```diff
@@ -35,7 +37,8 @@ describe("Queries", function () {
     ]);
   });
   test("Ethereum", async function () {
-    const ethereumQueries = new EthereumQueries();
+    const provider = new providers.JsonRpcProvider(process.env.NODE_URL_1);
+    const ethereumQueries = new EthereumQueries(provider);
     await Promise.all([
       ethereumQueries.getGasCosts("USDC"),
```

Comment:
Should we test that the return values are sensible values? For example, the gas costs are a wei value, decimals are 6, etc?